ArduPlane/quadplane.cpp

```diff
@@ -934,13 +953,13 @@ void QuadPlane::multicopter_attitude_rate_update(float yaw_rate_cds)
 
         if (use_yaw_target) {
             attitude_control->input_euler_angle_roll_pitch_yaw(plane.nav_roll_cd,
-                                                               plane.nav_pitch_cd,
+                                                               MAX(plane.nav_pitch_cd, q_fwd_nav_pitch_lim_cd),
```

Comment:
I think we have to set `nav_pitch_cd` to this limit before getting into this call. For example tailsitter transitions need to break these rules. I know that tailsitters should never have this set, but they might.